core/src/unprocessed_transaction_storage.rs

```diff
@@ -44,7 +48,7 @@ pub enum ThreadType {
 }
 
 #[derive(Debug, Default)]
-pub struct InsertPacketBatchesSummary {
+pub struct InsertPacketBatchSummary {
     pub(crate) num_dropped_packets: usize,
     pub(crate) num_dropped_gossip_vote_packets: usize,
     pub(crate) num_dropped_tpu_vote_packets: usize,
```

Comment:
might be cleaner to just replace these two fields with a `VoteBatchInsertionMetrics` since that struct has the same fields.

Alternatively we can have `InsertPacketBatchSummary` be an enum of

```
enum  InsertPacketBatchSummary {
      VoteBatchInsertionMetrics(metrics),
      PacketBatchMetrics(metrics),
}
```
and just expose a `fn total_dropped_packets()` and `fn total_dropped_tracer_packets()` method on the enum



```

Reply:
good idea also allows us to remove the (usize, usize) returns in unprocessed_packet_batches.